code/src/containers/category-section/CategorySection.scss

```diff
@@ -13,17 +13,24 @@
 
   &__element {
     display: flex;
-    justify-content: center;
     align-items: center;
     position: relative;
     flex-direction: column;
     gap: spacing(3);
   }
-  &__image {
+  &__link {
+    display: flex;
+    justify-content: center;
     max-width: 70%;
+    border-radius: 50%;
+  }
+  &__image {
+    max-width: 100%;
```

Comment:
Why did you use this style?

Reply:
I used it to make the AppLink component wrap perfectly around the round image.